src/3.0/types.ts

```diff
@@ -1,33 +1,42 @@
 // types generated by quicktype during postinstall phase
 import { OpenAttestationDocument as OpenAttestationDocumentV3 } from "../__generated__/schema.3.0";
-import { ProofPurpose, SchemaId, SignatureAlgorithm } from "../shared/@types/document";
+import { OpenAttestationHexString, ProofPurpose, SchemaId, SignatureAlgorithm } from "../shared/@types/document";
+import { Array as RunTypesArray, Record as RunTypesRecord, Static, String } from "runtypes";
 
 export interface Salt {
   value: string;
   path: string;
 }
+export const ObfuscationMetadata = RunTypesRecord({
+  obfuscated: RunTypesArray(OpenAttestationHexString)
+});
+export type ObfuscationMetadata = Static<typeof ObfuscationMetadata>;
 
-export interface VerifiableCredentialProofBase {
-  type: SignatureAlgorithm;
-  targetHash: string;
-  merkleRoot: string;
-  proofs: string[];
-  salts: string;
-  privacy: { obfuscated: string[] };
-  proofPurpose: ProofPurpose.AssertionMethod;
-}
+export const VerifiableCredentialProofBase = RunTypesRecord({
```

Comment:
Base? :eyes:

Reply:
initial name. want to change to the following ?
- VerifiableCredentialProofBase => VerifiableCredentialWrappedProof
- VerifiableCredentialProofSigned => VerifiableCredentialSignedProof

Reply:
ok